scripts/linters/other_files_linter.py

```diff
@@ -237,6 +241,36 @@ def check_webpack_config_file(self):
         return concurrent_task_utils.TaskResult(
             name, failed, error_messages, error_messages)
 
+    def check_strict_ts_config(self):
+        """Checks if the files in strict TS config are sorted
+        alphabetically.
+
+        Returns:
+            TaskResult. A TaskResult object representing the result of the lint
+            check.
+        """
+        name = 'Strict TS config'
+
+        failed = False
+        error_messages = []
+
+        strict_ts_config = json.load(python_utils.open_file(
+            STRICT_TS_CONFIG_FILEPATH, 'r'))
+
+        # Remove .ts extension from filepath for sorting.
+        files = [path[:-3] for path in strict_ts_config['files']]
```

Comment:
Do you even need to remove it from the path? It is at the end of the files and also it is the same for all the files.

Reply:
Suppose there are four files

```
some-file.ts,
some-file.spec.ts,
SomeFile.ts,
SomeFileSpec.ts
```

Sorting without removing the extension would result in

```
some-file.spec.ts,
some-file.ts,
SomeFile.ts,
SomeFileSpec.ts
```

You can notice the irregularity in spec files.

Reply:
Oh right, could you explain this in the comment above?

Reply:
done